TenantFile/Api/Extensions/SmsExtensions.cs

```diff
@@ -0,0 +1,39 @@
+﻿using Google.Cloud.Firestore;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+using TenantFile.Api.Models;
+using TenantFile.Api.Services;
+using Twilio.AspNet.Common;
+
+namespace TenantFile.Api.Extentions
+{
+    public static class SmsExtensions
+    {
+
+        public static async Task AddMessageAsync<T, TDoc>(this T message,
```

Comment:
I don't think we get a huge amount of benefit from making this an extension method. You can make this an instance method on SmsController, which has the benefit of exposing the documentDb so it doesn't have to have a passed in lambda.